subsys/bluetooth/controller/Kconfig

```diff
@@ -68,18 +71,28 @@ choice BT_LL_CHOICE
 	help
 	  Select the Bluetooth Link Layer to compile.
 
-config BT_LL_SW_SPLIT
+config BT_LL_SW_LLCP_LEGACY
 	bool "Software-based BLE Link Layer"
+	select BT_LL_SW
 	select BT_RECV_IS_RX_THREAD
```

Comment:
Not a fan of this construct. I would keep BT_LL_SW as is and then under the  "if BT_LL_SW" below i would add a BT_CTRL_LLCP_LEGACY config that is default y.

Reply:
Implemented as suggested